### PR TITLE
Adds feedback to the tachyon-doppler array's rotation

### LINKED ISSUE #28979: Cant Rotate Doppler array
Cant rotate it

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
tweak: The tachyon-doppler array's rotation now has messages, sprites, and examine text.
/:cl:

Fixes #28979.

So, you _can_ rotate the array, but before this PR it didn't actually _have any indication_ that it was rotated. The rotation would occur, but there were no messages or sounds of any kind to signify that it occurred. Because this is dumb, I have added messages, as well as directional sprites.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
The dish should also rotate.

**Comment by Xhuis** [comment]
I was iffy on that, as I might suck at making directional sprites, but I'll try nonetheless.

**Comment by Xhuis** [comment]
Yeah, I'm bad enough at spriting that any attempt at making the dish itself rotate would end in tragedy. The screen looks fine as-is; if anyone is willing to help with these sprites then feel free, but as it stands it would probably just look ugly if I attempted it.

**Comment by Repukan** [comment]
TRIPLE SIX FIVE FORKED TONGUE

**Comment by fizzle7** [comment]
We have directional sprites for the array on yogstation if you wanna take them or something.

**Comment by Xhuis** [comment]
After taking a look, I believe it's not in the 3/4 perspective, which people are pushing for. I'd have to get feedback from WJ on whether or not it's worth breaking the perspective.

File: code/game/machinery/doppler_array.dm [hunk 1]
```diff
@@ -45,7 +49,9 @@ GLOBAL_LIST_EMPTY(doppler_arrays)
 		return
 	if(usr.stat || usr.restrained() || !usr.canmove)
 		return
-	src.setDir(turn(src.dir, 90))
+	setDir(turn(src.dir, 90))
```
Comment (Cyberboss) [inline code/game/machinery/doppler_array.dm]: remove src

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/machinery/doppler_array.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/machinery/doppler_array.dm]: I forgot the second one, dang it.

**Review by Cyberboss — APPROVED** [review]

**Comment by WJohn** [comment]
Oh, good catch. These USED to have directions that reflected what was going on. That's an issue with the new sprites. Easy fix, you could pretty much copy paste the old dish sprites onto this and it'd work fine.

**Pull request merged**